ledger/ledger-api-test-tool/README.md

```diff
@@ -0,0 +1,15 @@
+## Suites
+
+### Command deduplication suites
```

Comment:
```suggestion
### Command deduplication test suites
```

Can you add a short intro to give context, e.g. why there are different suites for command deduplication in the first place?